src/redux/categories/categories.slice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    getStatus: () => 'Under construction',
+  },
+});
```

Comment:
- [ ] You   should update the state's categories property based on the value returned by the reducer. Example:

```
  reducers: {
    getStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },

```

Reply:
If you require any changes while reviewing, I would be grateful if you could connect with me on Zoom. This way, we can discuss the necessary modifications directly, saving time and avoiding any unnecessary delays. Your help in improving my work is greatly appreciated. Thank you.